spec/unleash/toggle_fetcher_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "spec_helper"
+
+RSpec.describe Unleash::ToggleFetcher do
+  describe '#save!' do
+    before do
+      Unleash.configure do |config|
+        config.url      = 'http://toggle-fetcher-test-url/'
+        config.app_name = 'toggle-fetcher-my-test-app'
+      end
+
+      fetcher_features = {
+        "version": 1,
+        "features": [
+          {
+            "name": "Feature.A",
+            "description": "Enabled toggle",
+            "enabled": true,
+            "strategies": [{ "name": "toggle-fetcher" }]
+          }
+        ]
+      }
+
+      WebMock.stub_request(:get, "http://toggle-fetcher-test-url/client/features").to_return(status: 200,
```

Comment:
Nit picking (so, not important): Could I recommend braking this into two lines? (for styling/legibility sake)

(.to_return) in a separate line.